bench/wtperf/runners/wtperf_run.sh

```diff
@@ -1,16 +1,23 @@
 #!/bin/bash
 
-# wtperf_run.sh - run wtperf regression tests on the Jenkins platform.
+# wtperf_run.sh - Reduce variance in wtperf measurements.
 #
-# The Jenkins machines show variability so we run this script to run
-# each wtperf test several times.  We throw away the min and max
-# number and average the remaining values.  That is the number we
-# give to Jenkins for plotting.  We write these values to a
-# test.average file in the current directory (which is 
-# build_posix/bench/wtperf).
+# Performance tests have a large variability from run to run, so we use this 
+# script to run each wtperf test several times. The script runs a user-defined 
+# number of times, drops the min and max values from each run, and reports the 
+# average of the remaining values to a results_$TESTNAME.txt file in the current 
+# directory. 
+# This script should be run from the same directory containing the wtperf binary.
 #
-# This script should be invoked with the pathname of the wtperf test
-# config to run and the number of runs.
+# Usage:
+# 	wtperf_run.sh test_conf numruns [wtperf_args]
+# 		test_conf:   The path the the wtperf configuration
+# 		rumruns:     How many times to run the test
```

Comment:
Is this meant to be numruns? Also, to remain consistent, could we change that to num_runs?

Reply:
Yes it is 👍 
I ended up changing the names of the other two arguments so that they match the variable names used in the script. Updated in d5ee691